src/js-context.ts

```diff
@@ -63,6 +63,14 @@ export default class JSContext {
     return methodNames;
   }
 
+  getClassName(): string | null {
```

Comment:
I see that you're using it in `validate-params`, so it seems fine....

More tests never hurt though.